nodes/src/nodes/llm_gmi/IGlobal.py

```
@@ -0,0 +1,136 @@
+                    if isinstance(data, dict):
+                        err = data.get('error')
+                        if isinstance(err, dict):
+                            etype = err.get('type')
+                            emsg = err.get('message') or data.get('message')
+                        else:
+                            etype = None
+                            emsg = data.get('message')
+                        message = self._format_error(status, etype, emsg, str(e))
+                    else:
+                        message = self._format_error(status, None, None, str(e))
+                except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)